src/CheckRightsService/CheckRightsService.csproj

```diff
@@ -14,4 +14,8 @@
     <ProjectReference Include="..\CheckRightsService.Validation\CheckRightsService.Validation.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="LT.DigitalOffice.Kernel" Version="1.1.9" />
```

Comment:
Probably, you should not write it here, as I know, we have targets in project, there are all packages. I may be wrong

Reply:
As I got it, this rule for all private fields (readonly too)

Reply:
fixed